package.json

```diff
@@ -13,6 +13,9 @@
     "tag": "lerna version --conventional-commits",
     "test": "lerna run test --stream"
   },
+  "dependencies": {
+    "focus-visible": "^5.1.0"
```

Comment:
Should this be a dev dependency? Consumers wouldn't need this in their `node_modules` to consume CSS.

Reply:
Just realized this is in the project root. This should also be pinned as well.

Reply:
oops, did a blind `yarn add`. Fixed.